content/about/agreements.md

```diff
@@ -133,7 +133,14 @@ Some links that might be useful to understand best practices for giving and rece
 
 Feedback for jovo can be provided directly in the #jovofeedback slack channel, including anonymous feedback.
 
+#### Contacting People Outside the Lab With Questions
 
+We agree that contacting people outside the lab with questions is effectively asking them to spend their limited time helping us. Since we would prefer to be the ones offering help rather than receiving it, we agree, prior to asking them for help, to:
+  1. Consider what it would take for us to solve to our problem (this means searching the literature, exploring and writing code, reading books, etc).
```

Comment:
```suggestion
  1. Consider what it would take for us to solve our problem (this means searching the literature, exploring and writing code, reading books, etc).
```